kafka_consumer/datadog_checks/kafka_consumer/config.py

```diff
@@ -27,6 +27,9 @@ def __init__(self, init_config, instance, log) -> None:
             if self._consumer_groups_regex
             else ""
         )
+        # https://github.com/confluentinc/confluent-kafka-python/issues/759
+        # Optimization to avoid OOM kill: AGENT-10389
```

Comment:
Let's not link private things:

```suggestion
        # Optimization to avoid OOM kill:
        # https://github.com/confluentinc/confluent-kafka-python/issues/759
```

Reply:
Updated.